### PR TITLE
Fix for vz-2246

### PR DESCRIPTION
**Pull request by skalyana-git**

### CODE REVIEW

File: bobs-books/bobbys-books/bobbys-coherence/src/main/java/org/books/bobby/AbstractLoader.java [hunk 1]
```diff
@@ -41,12 +41,14 @@ public Void process(ProcessingContext context, XmlElement xmlElement) throws Con
     @Override
     public void load(ConfigurableCacheFactory ccf) {
         NamedCache<String, Boolean> loaders = ccf.ensureCache("loaders", null);
+        InputStream is = null;
         if (loaders.lock(cacheName, 0)) {
             try {
```
Comment (markxnelson) [inline bobs-books/bobbys-books/bobbys-coherence/src/main/java/org/books/bobby/AbstractLoader.java:46]: this is ok, but we could probably just use a try with resources instead, and that would be a bit neater

**Review by markxnelson — COMMENTED** [review]
try with resources would be a lot cleaner

File: bobs-books/bobbys-books/bobbys-coherence/src/main/java/org/books/bobby/AbstractLoader.java (hunk 1 above)
Comment (skalyana-git) [reply bobs-books/bobbys-books/bobbys-coherence/src/main/java/org/books/bobby/AbstractLoader.java]: I have made the change to use try with resources. Can you please review? Thanks!

**Review by markxnelson — APPROVED** [review]

**Comment by markxnelson** [comment]
let me know when you merge this, i will need to publish updated images

**Comment by skalyana-git** [comment]
Hi Mark, I have merged the PR.

**Pull request merged**